### ISSUE TITLE
Missing shared libraris

### ISSUE THREAD

**Issue by foverzar** [boot2docker/boot2docker#914]
```sh
docker@boot2docker:/$ top
top: error while loading shared libraries: libncurses.so.5: cannot open shared object file: No such file or directory
```

**Comment by xavriley** [comment]
This looks to have been fixed recently. Are you using the latest version?

https://github.com/boot2docker/boot2docker/commit/135c7842a7f3871896c8b266a1af7de51f3a041c

**Comment by foverzar** [comment]
Yes, it was on the latest version then and I suppose it has something to do with a small fix I used to run unix commands through boot2docker ssh, as mysis ssh reports to use cygwin terminal, which is not supported in TinyCore. Anyway, right now switching $TERM envvar to any other supported terminal. Perhaps it should be considered to implement any kind of fix for this, because currently
```sh
boot2docker ssh
```
doesn't work correctly on stock install.
